hedera-node/hedera-app/src/main/java/com/hedera/node/app/grpc/impl/netty/NettyGrpcServerManager.java

```diff
@@ -184,14 +229,28 @@ public synchronized void start() {
         try {
             final var tlsPort = grpcConfig.tlsPort();
             logger.info("Starting TLS gRPC server on port {}", tlsPort);
-            nettyBuilder = builderFor(tlsPort, nettyConfig, profile);
+            nettyBuilder = builderFor(tlsPort, nettyConfig, profile, false);
             configureTls(nettyBuilder, nettyConfig);
-            tlsServer = startServerWithRetry(nettyBuilder, startRetries, startRetryIntervalMs);
+            tlsServer = startServerWithRetry(services, nettyBuilder, startRetries, startRetryIntervalMs);
             logger.info("TLS gRPC server listening on port {}", tlsServer.getPort());
         } catch (SSLException | FileNotFoundException e) {
             tlsServer = null;
             logger.warn("Could not start TLS server, will continue without it: {}", e.getMessage());
         }
+
+        if (grpcConfig.nodeOperatorPortEnabled()) {
+            try {
+                final var nodeOperatorPort = grpcConfig.nodeOperatorPort();
+                logger.info("Starting node operator gRPC server on port {}", nodeOperatorPort);
+                nettyBuilder = builderFor(nodeOperatorPort, nettyConfig, profile, true);
+                nodeOperatorServer =
+                        startServerWithRetry(nodeOperatorServices, nettyBuilder, startRetries, startRetryIntervalMs);
+                logger.info("Node operator gRPC server listening on port {}", nodeOperatorServer.getPort());
+            } catch (Exception e) {
+                nodeOperatorServer = null;
+                logger.warn("Could not start node operator gRPC server, will continue without it: {}", e.getMessage());
```

Comment:
Seems like most of this code is duplicated from above. Can be extracted to a method?